### PR TITLE
Feat(button-learnt-phrases): adds a disabled props and an action

### PR DESCRIPTION
**Pull request by voromahery**
Hello team,

I have just finished the tool button learnt phrases. If you have time, could you have a look at it, please.
Thank you in advance.

### CODE REVIEW

**Review by constantingoeldel — APPROVED** [review]

**Comment by constantingoeldel** [comment]
Good naming of pull request and commits!

File: src/screens/AddNewTerm.js [hunk 1]
```diff
@@ -21,15 +21,15 @@ export default ({navigation, categories, addNewPhrase}) => {
   const [addEnglishPhrase, setAddEnglishPhrase] = useState('');
   const [addMalagasyPhrase, setAddMalagasyPhrase] = useState('');
 
-  const isButtonEnable = addEnglishPhrase === '' || addMalagasyPhrase === '' || selectedCategory === '';
+  const isButtonEnable =
```
Comment (constantingoeldel) [inline src/screens/AddNewTerm.js:24]: This does not seem to be part of the learnt phrases user story and therefore should not be part of the PR

**Review by Mickaellah — APPROVED** [review]

**Review by bakozetra — APPROVED** [review]
